src/installer/steps/configure/chroot_conf.rs

```diff
@@ -182,6 +183,10 @@ impl<'a> ChrootConfigurator<'a> {
             self.chroot.command("passwd", &[user]).stdin_input(&pass).run()?;
         }
 
+        let dest = self.chroot.path.join(&["home/", user, "/.profile"].concat());
```

Comment:
I don't understand how this works. $HOME/.profile is a shell file, this is copying the icon to that path. Am I missing something?

Reply:
I have the wrong file. It's `.face`